common/junit/src/main/java/org/eclipse/dataspaceconnector/common/junit/testextensions/EdcRuntimeExtension.java

```diff
@@ -40,11 +40,12 @@
 import static org.eclipse.dataspaceconnector.boot.system.ExtensionLoader.loadMonitor;
 
 /**
- * A JUnit extension for running an embedded EDC runtime as part of a test fixture.
- * A custom gradle task printClasspath is used to determine the runtime classpath of the module to run.
- * The runtime obtains a classpath determined by the Gradle build.
+ * A JUnit extension for running an embedded EDC runtime as part of a test fixture. A custom gradle task printClasspath
```

Comment:
I don't get all these mowed line returns, they are useful to keep the text in a readable manner, I would keep them

Reply:
i think this may have been done by some autoformatter.